src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java

```diff
@@ -29,6 +29,12 @@ void extractURLFormLink() {
                 formatter.format("away.php?to=http%3A%2F%2Fwikipedia.org&a=snippet"));
     }
 
+    @Test
+    void validURLUnmodified() {
```

Comment:
We try to gradually migrate to Google's casing rules: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Thus, please introduce defined camel case

```suggestion
    void validUrlUnmodified() {
```